examples/upload-file-via-browser/package.json

```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "babel-core": "^5.4.7",
     "babel-loader": "^5.1.2",
-    "ipfs-api": "^12.1.7",
+    "ipfs-api": "../../",
```

Comment:
Hm... 🤔

Reply:
That's an example in the examples directory. Just repointed it to use the current ipfs-api rather than a published version.

Reply:
Thanks, now it makes sense 👍